specification/quota/resource-manager/Microsoft.Quota/preview/2021-03-15-preview/quota.json

```diff
@@ -34,16 +34,118 @@
     }
   },
   "paths": {
-    "/{scope}/providers/Microsoft.Quota/quotaLimits/{resourceName}": {
+    "/{scope}/providers/Microsoft.Quota/usages/{resourceName}": {
+      "get": {
+        "tags": [
+          "UsagesInformation"
+        ],
+        "description": "Gets the current usage of a resource.",
+        "operationId": "Usages_Get",
+        "x-ms-examples": {
+          "Quotas_UsagesRequest_ForNetwork": {
+            "$ref": "./examples/getNetworkOneSkuUsages.json"
+          },
+          "Quotas_UsagesRequest_ForCompute": {
+            "$ref": "./examples/getComputeOneSkuUsages.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceNameInParameters"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeInPath"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The usages information will be returned in the response.",
+            "schema": {
+              "$ref": "#/definitions/CurrentQuotaLimitBase"
```

Comment:
Why does usages return the "currentValue" in a property named "limit"? That isn't user friendly and doesnt seem necessary

Reply:
I will change it to "usages" - I will the polymorphism for future extension of this property, when the usages values are combined per SKU family, rather that individual SKU.
"usages": {
            "limitObjectType": "UsagesValue",
            "value": 10
          },

Reply:
Separated out quotas and usages.

Reply:
> Separated out quotas and usages. usages will be listed under "usages" object, with optional field -
"UsagesTypes": {
      "description": "The quota or usages limit types.",
      "enum": [
        "Individual",
        "Combined"
      ],